static/bidder-info/liftoff.yaml

```diff
@@ -0,0 +1,10 @@
+endpoint: "https://rtb.ads.vungle.com/bid/t/c770f32"
+maintainer:
+  email: platform-ssp@liftoff.io
```

Comment:
sent email to this email address for verification.

Reply:
@Vungle-GordonTian email is bounced. Could you please re-check if this email ID is working.

Reply:
@gargcreation1992 I'm sure this is the valid and correct email address. liftoff.io is blocked only in China, otherwise it should be ok. Can u provide your email info such as address and send server?

Reply:
Here is the reply that we have received - 
```
Hello [prebid@microsoft.com](mailto:prebid@microsoft.com),

We're writing to let you know that the group you tried to contact (platform-ssp) may not exist, or you may not have permission to post messages to the group. A few more details on why you weren't able to post:

 * You might have spelled or formatted the group name incorrectly.
 * The owner of the group may have removed this group.
 * You may need to join the group before receiving permission to post.
 * This group may not be open to posting.
 ```

Reply:
@gargcreation1992 Many thanks for the detailed info. I'll refer to our IT to find out the root cause.

Reply:
@gargcreation1992 I've reach out to our IT and they have now adjusted the permissions for [platform-ssp@liftoff.io](mailto:platform-ssp@liftoff.io) to accept incoming emails from external senders.

Could you try sending an email again and it should all be good.

Reply:
done. @gargcreation1992